Milliner/src/Service/MillinerService.php

```diff
@@ -542,4 +543,40 @@ public function saveExternal(
         // Return the response from Fedora.
         return $response;
     }
+
+    /**
+     * Creates a new LDP-RS in Fedora from a Node.
+     * {@inheritDoc}
+     */
+    public function createVersion(
+        $uuid,
+        $token = null
+    ) {
+        $urls = $this->gemini->getUrls($uuid, $token);
+        $fedora_url = $urls['fedora'];
```

Comment:
Maybe an edge case, but if something is deleted it should disappear from Gemini. A subsequent request will return `null`, probably want to do something like
https://github.com/Islandora/Crayfish/pull/74/files#diff-a3c5561a4953e4787761a32c6ce42b77R471
and throw an error if the item is gone.

Reply:
Ok, what would it return if $urls is empty? like a `new Response(404)`?

Reply:
if this is the right approach, can you verify the test accurately tests this?